pylibdmtx/pylibdmtx.py

```diff
@@ -251,3 +253,69 @@ def decode(image, timeout=None, gap_size=None, shrink=1, shape=None,
                                 break
 
     return results
+
+
+@contextmanager
+def libdmtx_encoder():
+    encoder = dmtxEncodeCreate()
+    if not encoder:
+        raise PyLibDMTXError('Could not create encoder')
+
+    try:
+        yield encoder
+    finally:
+        dmtxEncodeDestroy(byref(encoder))
+
+
+def encode(data, scheme='ascii', symsize=None):
```

Comment:
With the change I suggest at the top of this module, alter `scheme='Ascii'`. Could we also default `symsize='ShapeAuto'`? I haven't played with the 'auto' options so am not sure how they work.